src/PlatformEngines.jl

```diff
@@ -888,19 +365,10 @@ end
 Compress `src_dir` into a tarball located at `tarball_path`.
 """
 function package(src_dir::AbstractString, tarball_path::AbstractString)
```

Comment:
This is also test only, right?

Reply:
Not sure, I thought the Artifacts code used it too, but I could be wrong.